iptables.yaml

```diff
@@ -70,6 +71,18 @@ pipeline:
       install -D -m644 ebtables.confd "${{targets.destdir}}"/etc/conf.d/ebtables
 
 subpackages:
+  - name: iptables-privileged
```

Comment:
the patch is in draft mode because we can discuss more on the package name here. I was thinking if we can name it as `iptables-linkerd-compat` similar to our bitnami variant of package where we name a package as `*-bitnami-compat`

Reply:
I'd call it what it is - something like `iptables-cap_set_admin`. Then it can be used for any package or image that needs it, not just linkerd